en/index.html

```diff
@@ -122,16 +122,16 @@
         <div class="container">
             <div class="row">
                 <div class="col-lg-12 text-center">
-                    <h2>Présentation</h2>
+                    <h2>Presentation</h2>
                     <hr class="star-primary">
                 </div>
             </div>
             <div class="row">
                 <div class="col-lg-4 col-lg-offset-2">
-                    <p>REGOVAR est un projet collaboratif, libre et ouvert de logiciel d’analyse de données de séquençage haut débit pour les panels de gènes, l’exome et le génome (DPNI, recherche de SNV, CNV, SV...). Il vise à impliquer et fédérer les différentes communautés concernées (généticiens cliniciens, biologistes et bioinformaticiens), sans limites institutionnelles ou géographiques. Il se base exclusivement sur des technologies et des logiciels libres et gratuits, éliminant ainsi toute contrainte contractuelle et budgétaire.</p>
+                    <p>REGOVAR is a collaborative, free and open-source project dealing with the analysis of high throughput sequancing data for the gene panels, the exome and the genome (NIPT, discovery of SNV, CNV, SV...). It aims to engage and federate the various concerned communities (clinical geneticists, biologists and bioinformaticians), without institutional and geographical boundaries. It is exclusively based on free and open-source technologies and software, removing any contractual and budget constraints.</p>
                 </div>
                 <div class="col-lg-4">
-                    <p>REGOVAR est un logiciel permettant le traitement de données génétiques, depuis la récupération des fichiers produits par les séquenceurs, quelle qu’en soit la technologie, jusqu’à la génération de rapports illustrés et de comptes-rendus d’analyse en passant par les contrôles de qualité, la détection, l’annotation, le filtrage, la priorisation et la visualisation de variants.</p>
+                    <p>REGOVAR is a software program allowing the processing of genetic data, from the retrieval of files which are procuded by sequencers, regardless of the technology, to the generation of illustrated reports and accounts with the quality control, the screening, the annotation, the filtering, the prioritisation and the visualisation of the variants.</p>
```

Comment:
procuded → produced

Reply:
Thanks ! Good catch, i've corrected it ;).